edge/pkg/common/util/config.go

```diff
@@ -78,7 +77,19 @@ func GenerateTestYaml(test interface{}, path, filename string) error {
 	if err != nil {
 		return err
 	}
-	err = ioutil.WriteFile(path+"/"+filename+".yaml", data, 0777)
+	fp, err:=os.Create(path+"/"+filename+".yaml")
+	if err != nil {
+		return err
+	}
+	_,err=fp.Write(data)
```

Comment:
Please run go fmt against this file to fix the go fmt issues.

Reply:
oops, my bad. Done, thanks :)